libs/vkd3d/command.c

```diff
@@ -5557,25 +5557,17 @@ static void d3d12_command_list_update_descriptors(struct d3d12_command_list *lis
 {
     struct vkd3d_pipeline_bindings *bindings = &list->pipeline_bindings[bind_point];
     const struct d3d12_root_signature *rs = bindings->root_signature;
+    const struct d3d12_bind_point_layout *bind_point_layout;
     VkPipelineBindPoint vk_bind_point;
     VkShaderStageFlags push_stages;
     VkPipelineLayout layout;
 
     if (!rs)
         return;
 
-    if (list->active_pipeline_type == VKD3D_PIPELINE_TYPE_RAY_TRACING)
-    {
-        /* We might have to emit to RT bind point,
-         * but we pretend we're in compute bind point. */
-        layout = bindings->rt_layout.vk_pipeline_layout;
```

Comment:
Nice. This seems to actually fix a bug since we always used `bindings->layout` instead of `rt_layout` in vkCmdPushConstants above ... Wonder why this didn't explode ...